.github/workflows/tests_and_release.yml

```
@@ -0,0 +1,68 @@
+name: Validate and Release connector
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Tableau Connector SDK
+        uses: actions/checkout@v4
+        with:
+          repository: tableau/connector-plugin-sdk
+          ref: tableau-2024.2
+
+      - name: Checkout CrateDB connector
+        uses: actions/checkout@v4
+        with:
+          path: cratedb-tableau-connector
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Set up Tableau connector packager
+        working-directory: connector-packager
+        run: |
+          python -m venv .venv
+          source ./.venv/bin/activate
+          python -m pip install setuptools
+          python setup.py install
```

`setup.py install` is deprecated.
```suggestion
          python -m pip install .
```